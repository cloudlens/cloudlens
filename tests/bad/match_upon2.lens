match(entry) upon (hi) {
    "(?<foo>foo)"
}
