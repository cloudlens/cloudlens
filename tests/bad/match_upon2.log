[
{
"hi":      "Bonjour",
"message": "6 pommes",
"foo":     {"a":3}
},
{
"hi":      "Hello",
"message": "2 apples",
"bar":     {"a":{"b":5}}
},
{
"test":42
}
]
