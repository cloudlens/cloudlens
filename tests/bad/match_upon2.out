Invalid path: hi. Maybe you forgot the entry name.
